share/rocm/cmake/ROCMConfig.cmake

```diff
@@ -1,6 +1,9 @@
 # ######################################################################################################################
-# Copyright (C) 2017 Advanced Micro Devices, Inc.
+# Copyright (C) 2023 Advanced Micro Devices, Inc.
 # ######################################################################################################################
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
-include(ROCMChecks)
+message(INFO
```

Comment:
This should be of type [`DEPRECATION`](https://cmake.org/cmake/help/v3.0/command/message.html#general-messages).

Reply:
Thanks for the correction, updated.